pkg/awsManager/vpcVpnManager.go

```diff
@@ -350,19 +333,16 @@ func DeleteSubnetsForVPC(client clientpkg.Client, vpcId *string) error {
 			break
 		}
 	}
-
 	return nil
 }
 
-func DeleteRouteTables(client clientpkg.Client, vpcId *string) error {
-
+func deleteRouteTables(client clientpkg.Client, vpcId *string, logger logr.Logger) error {
```

Comment:
why was this unexported?

Reply:
because it is a helper method that isn't and shouldn't be used outside this file